schutzbot/deploy.sh

```diff
@@ -32,27 +30,15 @@ enabled=1
 gpgcheck=0
 # Default dnf repo priority is 99. Lower number means higher priority.
 priority=5
-
-[osbuild-composer]
-name=osbuild-composer ${OSBUILD_COMPOSER_COMMIT}
-baseurl=${DNF_REPO_BASEURL}/osbuild-composer/${DISTRO_VERSION}/${ARCH}/${OSBUILD_COMPOSER_COMMIT}
-enabled=1
-gpgcheck=0
-# Give this a slightly lower priority, because we used to have osbuild in this repo as well.
-priority=10
 EOF
 
 if [[ $ID == rhel || $ID == centos ]] && ! rpm -q epel-release; then
     # Set up EPEL repository (for ansible and koji)
     sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${VERSION_ID%.*}.noarch.rpm
 fi
 
-# Install the Image Builder packages.
-# Note: installing only -tests to catch missing dependencies
-sudo dnf -y install osbuild-composer-tests
-
-# Set up a directory to hold repository overrides.
-sudo mkdir -p /etc/osbuild-composer/repositories
+sudo dnf -y install osbuild osbuild-luks2 osbuild-lvm2 osbuild-ostree
 
-# Temp fix until composer gains these dependencies
-sudo dnf -y install osbuild-luks2 osbuild-lvm2
+# workaround for centos 8 where python-osbuild is not found.
```

Comment:
Is this workaround still needed? Also, how does it work?

Reply:
So this, is a complete mystery. On centos8 when you install `osbuild`, even if it correctly pulls `python-osbuild` as a dependency it will say to you that the module `osbuild` does not exist. And that's true on `manifest-db` too. So as I was trying to debug that the other, I installed pip to list the modules installed, and it started to work. I really thought "what the heck ???" and then I moved on without answering the issue because I really thought it would be a rabbit hole and I wanted to finish my current task instead.

It's bothering me still a bit, because, when you install `osbuild` as a dependency of `composer` you never run into this issue. So there might be a dependency missing in `osbuild`, but, which one ? I think we can answer that in a follow up.

Reply:
Also, this issue is not reproducible on a centos8 machine that you would reserve directly on aws. I did try to reserve a machine, same flavor as in the CI to experiment with that. Installing osbuild did not gave me an issue. So there's something missing, and I don't know what it is.

Reply:
We can continue this discussion in the follow up.